openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/whatsNewData.ts

```diff
@@ -1347,4 +1349,192 @@ export const WHATS_NEW = [
+      },
+      {
+        title: 'Custom Theme',
+        description:
+          "OpenMetadata previously supported uploading your company logo, monogram, and favicon to customize the platform's appearance according to your brand identity. Now, you can take it a step further by customizing the theme with colors that perfectly align with your company's branding.",
+        isImage: false,
+        path: 'https://www.youtube.com/embed/-NiU1flBHs0',
+      },
+      {
+        title: 'Data Quality Filters',
+        description:
+          'We have improved the filters for data quality. Now you have additional filtering options for test suites and test cases.',
+        isImage: false,
+        path: 'https://www.youtube.com/embed/UNOHvBMVcYM',
+      },
+      {
+        title: 'Data Profiler',
+        description:
+          'A global profiler configuration page has been implemented for the data profiler. This allows Admins to exclude certain metric computations for specific data types. Navigate to Settings > Preferences > Profiler Configuration to define the metrics to compute based on column data types.',
+        isImage: true,
+        path: profilerConfigPage,
+      },
+      {
+        title: 'Incident Manager',
+        description:
+          'Based on the latest failed test cases, a sample of failed rows will be displayed in the Incident Manager. Users can quickly verify the cause of failure based on this sample data. The failed sample data will be deleted once the issue is resolved. This is a Collate only feature.',
+        isImage: true,
+        path: incidentManagerSampleData,
+      },
+    ],
+    changeLogs: {
+      ['Backward Incompatible Changes']: `     
+Tooling:
+-   Metadata Backup/Recovery is deprecated. No further support will be provided.
+-   Users are advised to use database native tools to backup and store it in their object store for recovery.
+-   bootstrap/bootstrap_storage.sh has been deprecated in favor of bootstrap/openmetadata-ops.sh
+
+UI:
+
+-   Activity has been improved. New update specific cards display critical information such as data quality test case updates, description, tag update or removal.
+-   For Lineage, the Expand All button has been removed. A new Layers button is introduced at the bottom left corner. With the Layers button, you can add Column Level Lineage or Data Observability details to your Lineage view.
+-   View Definition is now renamed to Schema Definition.
+-   Adding Glossary Term view is improved. Now we show glossary terms hierarchically enabling a better understanding of how the terms are setup while adding it to a table or dashboard.
+-  For Classification, users can set classification to be mutually exclusive only at the creation time. Once created, you cannot change it back to mutually non-exclusive or vice-versa. This is to prevent conflicts of adding multiple tags that belong to same classification and later turning the mutually exclusive flag back to true.
```

Comment:
For Classification, users can set classification to be mutually exclusive only `**at the time of creation**`. Once created, you cannot change it back to mutually non-exclusive or vice-versa. This is to prevent conflicts of adding multiple tags that belong to same classification and later turning the mutually exclusive flag back to true.